packages/ui/core-components/src/lib/unsorted/viz/references/ReferenceLine/_ReferenceLine.svelte

```diff
@@ -1,271 +0,0 @@
-<script context="module">
-	export const evidenceInclude = true;
-</script>
-
-<script>
-	import { getContext } from 'svelte';
-	import { propKey, configKey } from '@evidence-dev/component-utilities/chartContext';
-	import { formatValue } from '@evidence-dev/component-utilities/formatting';
-	import checkInputs from '@evidence-dev/component-utilities/checkInputs';
-	import ErrorChart from '../../core/ErrorChart.svelte';
-	import { uiColours } from '@evidence-dev/component-utilities/colours';
-
-	let props = getContext(propKey);
-	let config = getContext(configKey);
-
-	export let x = undefined;
-	export let y = undefined;
-	export let x2 = undefined;
-	export let y2 = undefined;
-	export let data = undefined;
-	export let label = undefined;
-
-	export let color = undefined;
-	export let lineColor = undefined;
-	export let labelColor = undefined;
-	export let lineWidth = undefined;
-	export let lineType = 'dashed'; // solid, dashed, or dotted
-	// Override arrow symbol with custom svg path which is a nice aspect triangle
-	let arrowPath = 'path://M0,10 L5,0 L10,10 z';
-	export let symbol = 'none';
```

Comment:
This prop has been removed which is a breaking change - it was a symbol appearing at the end of the line

(The default arrow was nasty so I replaced it, open to changing this, but we should definitely still have some kind of support for the symbol)

Reply:
I had removed this prop because it wasn't working properly and I thought it was just some leftover code

https://github.com/evidence-dev/evidence/blob/reference-line-symbol-doesnt-work/packages/ui/core-components/src/lib/unsorted/viz/references/ReferenceLine/ReferenceLine.stories.svelte#L52

No symbol is rendered anywhere

![image](https://github.com/user-attachments/assets/aee93c13-e175-4042-a97c-4c708fc54db5)

I can re-add and fix this functionality

Reply:
The symbol is only rendered at the end of the line when you specify a y2 point - allows you to use as an arrow

Reply:
Added support and docs for `symbol`, `symbolStart`, and `symbolEnd` here
[c91ba01](https://github.com/evidence-dev/evidence/pull/2184/commits/c91ba010a41088b72265f4f4a3c01b40030566e3)